packages/loader/container-loader/src/container.ts

```diff
@@ -178,6 +178,10 @@ export async function waitContainerToCatchUp(container: IContainer) {
         };
         container.on("closed", closedCallback);
 
+        // Depending on config, transition to "connected" state may include the guarantee
+        // that all known ops have been processed.  If so, we may introduce additional wait here.
```

Comment:
What do you all think - is this correct or not?  I may not add additional delay since events fire synchronously.